packages/apps/web-viewer-test/src/components/home/IModelBankHome.tsx

```diff
@@ -106,8 +106,8 @@ export const IModelBankHome: React.FC = () => {
     <div style={{ height: "100vh" }}>
       <Viewer
         authClient={authClient}
-        iTwinId={iTwinId}
-        iModelId={iModelId}
+        iTwinId={iTwinId ?? ""}
```

Comment:
why the need for empty strings? Can't be left undefined?

Reply:
because of the xor, these args are now required, and process.env.VAR is potentially undefined